x/compute/client/rest/tx.go

```diff
@@ -113,12 +117,18 @@ func instantiateContractHandlerFn(cliCtx context.CLIContext) http.HandlerFunc {
 			return
 		}
 
+		//res, err := GetCodeHashByCodeId(cliCtx, string(codeID))
+		//if err != nil {
+		//	return
+		//}
+
 		msg := types.MsgInstantiateContract{
-			Sender:    cliCtx.GetFromAddress(),
-			Code:      codeID,
-			InitFunds: req.Deposit,
-			InitMsg:   req.InitMsg,
-			Admin:     req.Admin,
+			Sender:            cliCtx.GetFromAddress(),
+			Code:      		   codeID,
+			CallbackCodeHash:  "",
```

Comment:
again, why is this empty?

Reply:
Because this is just a regular message and not a callback, so the code hash is already contained in the encrypted message